packages/docs/introduction.yaml

```diff
@@ -25,60 +25,57 @@ _ref:
         type: Markdown
         properties:
           content: |
-            ##### TODO: This page needs to be updated.
+            Lowdefy makes is really easy to build web apps. With Lowdefy you can build CRUD apps, admin panels, internal tools, BI dashboards, workflow apps, websites, blogs, webforms or even documentation pages like this one.
 
-
-            Lowdefy simplifies creating software that helps manage internal data - think web apps, admin panels, BI dashboards, CRUD and workflow apps. Within most companies the need exists to connect to a variety of data sources and execute actions on such data. This must be done without compromising data security while respecting complex data integrity rules.
-
-            Lowdefy aims to enable companies to better control and monitor business data, by simplifying the process of creating internal tools. Lowdefy makes these tools easy to build, change and maintain, making it easier to evolve your internal tools with your business.
+            Lowdefy apps are defined in YAML config that follows a structured schema - this makes Lowdefy apps are easy to read, write and understand.
```

Comment:
...this makes Lowdefy apps ~are~ easy to read, write and understand...

Reply:
Fixed